src/assets/usage-examples-code/sprites/sprite_set_x/sprite_set_x-8-sprite-x.txt

```diff
@@ -0,0 +1,3 @@
+### sprite_set_x() usage example
+
+The following code is an example of using sprite set x which sets the x position of a sprite on a splashkit window.
```

Comment:
Update this to be a bit clearer and use capital first letters on the function name with a proper link.